src/Model/Split.java

```diff
@@ -0,0 +1,16 @@
+package Model;
+
+public class Split {
+
+  public static void getSplitWords() {
```

Comment:
I asume this class will be used to split input text into words, if so then you should make it accepts an input parameter(the text) and return the splited words.